extensions/protobuf/build.gradle

```diff
@@ -10,23 +10,25 @@ dependencies {
     api 'com.google.protobuf:protobuf-java:3.25.3'
```

Comment:
worth updating this to point to our One True Version now?

Reply:
Definitely. Missed this one.